conf.d/electrum

```diff
@@ -1,22 +1,35 @@
 #!/bin/bash -e
 
 SRC=/usr/local/src
-PATCHES=$SRC/electrum-patches
+PATCHES=$SRC/electrum.patch
 
 # install electrum deps
 easy_install $SRC/ecdsa-*.tar.gz
 easy_install $SRC/slowaes-*.tar.gz
-rm $SRC/{ecdsa-*.tar.gz,slowaes-*.tar.gz}
+easy_install $SRC/jsonrpclib-*.tar.gz
```

Comment:
I know that the precedent for including this stuff in the overlay was set by Alon & Liraz, IMO it's not ideal. Where do these files come from? Perhaps we should be checking them against published SHA sums somewhere. If you can give me the urls of where they come from I can take care of that.